SConstruct

```diff
@@ -60,14 +64,10 @@ The following environment variables modify the build environment:
     OPENSSL_ROOT
       Path to the openssl directory.
 
-The following extra options may be used:
-    --ninja         Generate a `build.ninja` build file for the specified target
-                    (see: https://martine.github.io/ninja/). Only gcc and clang targets
-                     are supported.
+"""
```

Comment:
It isn't clear how to do common tasks with this SConstruct - for example, that most of the things a user wants to edit live in `bin/python/ripple/build/module/__init__.py`. I went through all the commits made to SConstruct since 2014 and made a list of common tasks. We should look at that list and make sure there's a doc explaining how to do the common tasks (unless it's obvious how to do them). Doc:  https://ripplelabs.atlassian.net/wiki/display/~determan/Typical+SConstruct+Tasks